local_config.yaml

```diff
@@ -10,5 +10,5 @@ observability:
 #   1. enabled: true
 #   2. path: {PATH_TO_LOCAL_HELM_CHART}
 helm_chart_local_repo:
-  enabled: false
+  enabled: true
```

Comment:
Please add a TODO_IN_THIS_PR to revert this after the upstream changes are merged.

Reply:
This is still false and no TODO_IN_THIS_PR was added